modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Detail.cshtml.cs

```diff
@@ -79,6 +86,8 @@ private async Task GetData()
         {
             Blog = await _blogAppService.GetByShortNameAsync(BlogShortName);
             Post = await _postAppService.GetForReadingAsync(new GetPostInput { BlogId = Blog.Id, Url = PostUrl });
+            PostsList = await _postAppService.GetListByUserIdAsync(Post.Writer.Id);
+            LatestPosts = (await _postAppService.GetListByBlogIdAndTagNameAsync(Blog.Id, TagName)).Items;
```

Comment:
- [ ] Limit to number instead of fetching all posts.
- [ ] Is there a guarantee that you will bring the latest posts? Otherwise, write a new method